golang/cosmos/x/swingset/testing/queue.go

```diff
@@ -13,13 +14,15 @@ import (
 func GetActionQueueRecords(t *testing.T, ctx sdk.Context, swingsetKeeper keeper.Keeper) ([]string, error) {
 	vstorageKeeper := keeper.GetVstorageKeeper(t, swingsetKeeper)
 	actionQueueName := keeper.StoragePathActionQueue
-	return vstoragetesting.GetQueueItems(ctx, vstorageKeeper, actionQueueName)
+	// TODO: fix this type assertion vstorage needs interface needs to be updated
```

Comment:
Do you plan on implementing this TODO?  If not:

```suggestion
	// TODO: fix this type assertion, vstorage needs interface to be updated
```